packages/preact-iso/lazy.d.ts

```diff
@@ -0,0 +1,5 @@
+import { ComponentChildren, VNode } from 'preact';
+
+export default function lazy<T>(load: () => Promise<{ default: T }>): T;
```

Comment:
The `.default` property is optional and only used if present, I think this should do the trick:

```suggestion
export default function lazy<T>(load: () => Promise<{ default: T } | T>): T;
```